otherlibs/unix/unixLabels.mli

```diff
@@ -1001,20 +1266,28 @@ val getlogin : unit -> string
 (** Return the login name of the user executing the process. *)
 
 val getpwnam : string -> passwd_entry
-(** Find an entry in [passwd] with the given name, or raise
-   [Not_found] if the matching entry is not found. *)
+(** Find an entry in [passwd] with the given name.
+   @raise Not_found if no such entry exist.
+
+   On Windows, always raise [Not_found]. *)
```

Comment:
The `@raise` needs to be after the Windows comment (and `getgrnam`, `getpwuid`, `getgrgid`) below. Although for these perhaps `@raise Not_found on Windows (always), or if no such entry exists.` is better.

Reply:
Fixed.